Oracle/diagnosis.sql

```diff
@@ -1,7 +1,5 @@
 /** diagnosis - create and populate the diagnosis table.
 */
-insert into cdm_status (task, start_time) select 'diagnosis', sysdate from dual
```

Comment:
was this just comment or luigig magic hint

Reply:
answered above